src/libtorch.cc

```diff
@@ -79,6 +80,9 @@ class ModelState : public BackendModel {
       std::unique_ptr<torch::jit::script::Module>* torch_model);
 
   bool EnabledOptimizedExecution() { return enable_optimized_execution_; }
+  bool EnabledTensorExprFuser() { return enable_tensor_fuser_; }
+  bool EnabledJitProfilingMode() { return enable_jit_profiling_; }
```

Comment:
A mode symbolizes it will have a non-binary value but in this case it is a flag instead. Shouldn't `EnabledJitProfiling()` be enough?
Similarly for EnabledJitExecutorMode.